src/js/utils/strings.ts

```diff
@@ -40,23 +40,24 @@ export function extension(path) {
     if (path.lastIndexOf('.') > -1) {
         return path.substr(path.lastIndexOf('.') + 1, path.length).toLowerCase();
     }
+    return '';
 }
 
 // Convert seconds to HH:MN:SS.sss
-export function hms(secondsNumber) {
+export function hms(secondsNumber: number): string {
     const h = (secondsNumber / 3600) | 0;
     const m = ((secondsNumber / 60) | 0) % 60;
     const s = secondsNumber % 60;
-    return pad(h, 2) + ':' + pad(m, 2) + ':' + pad(s.toFixed(3), 6);
+    return pad(h.toString(), 2) + ':' + pad(m.toString(), 2) + ':' + pad(s.toFixed(3), 6);
 }
 
 // Convert a time-representing string to a number
-export function seconds(str, frameRate) {
+export function seconds(str?: string | number, frameRate?: number): number {
     if (!str) {
         return 0;
     }
     if (isValidNumber(str)) {
-        return str;
+        return parseFloat(str.toString());
```

Comment:
hmmm, if we type the return value of `isValidNumber` do we still need this `parse/toString` set of calls?

Reply:
Nope! Not needed. Not sure why I did that.